src/thunderbots/software/ai/passing/pass_generator.h

```diff
@@ -249,10 +249,6 @@ namespace AI::Passing
          */
         std::vector<Pass> generatePasses(unsigned long num_passes_to_gen);
 
-        // This constant is used to prevent division by 0 in our implementation of Adam
-        // (gradient descent)
-        static constexpr double eps = 1e-8;
```

Comment:
Why did this get removed?

Reply:
This was originally used for gradient descent, but we moved it over when we broke out gradient descent into it's own class.